changelog.d/11340.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug, introduced in Synapse 1.46.0, which caused some callbacks in legacy password authentication provider modules to not be registered. Modules using the generic module API were not affected.
```

Comment:
Is there any more information about *which* callbacks? How would a sysadmin know if they were affected?

Reply:
Added